src/openfl/_internal/text/TextEngine.hx

```diff
@@ -213,6 +213,16 @@ class TextEngine {
 		}
 	}
 
+	function getWordSeparatorIndex(startIndex:Int):Int {
```

Comment:
I would likely unify the `getWordSeparatorIndex` and `getLineBreakIndex` and the way the return if/else statements are written.
You could also unify the default parameter of `startIndex` to twice be `0` or twice not set?

Reply:
Yep, I also noticed that it would be nice to do this 👍